libs/sr/formula/src/generators/gen-sheet/char/__sheet__.ts.template

```diff
@@ -9,41 +9,15 @@ import {
   own,
   ownBuff,
   register,
+  registerBuff,
   teamBuff,
-} from '../util'
-import { dmg, entriesForChar, getBaseTag, scalingParams } from './util'
+} from '../../util'
+import { dmg, entriesForChar, getBaseTag } from '../util'
 
 const key: CharacterKey = '<%= sheet %>'
 const data_gen = allStats.char[key]
+const dm = mappedStats.char[key]
```

Comment:
I wonder if it makes sense to do `mappedStats.char<%= sheet %>` so that we get benefit of intelliji autocompletes?

Reply:
This dm obj comes fully typed according to the char already.
So `dm.basic[0]` will have a type and be completely safe. Is that what you meant?

Reply:
I was referring being able to have IDE auto complete as dev experience

Reply:
Those work in the current experience